internal/worker/cmd_meta.go

```diff
@@ -83,6 +109,35 @@ var CommandsMeta = map[string]CmdMeta{
 		CmdType: SingleShard,
 	},
 
+	// Multi-shard commands.
```

Comment:
We may want to have a separate category for multi-step-multi-shard commands.

Reply:
+1